### PR TITLE
feat: add flashstart

### PR DESCRIPTION
**Pull request by nicolorevelli**
Add FlashStart page

Card: https://trello.com/c/ZkkStjuf/138-flashstart-ui

### CODE REVIEW

File: public/i18n/en/translation.json [hunk 1]
```diff
@@ -560,6 +560,24 @@
...
     },
+    "flashstart": {
+      "title": "Flashstart filter",
+      "content_title": "Settings",
+      "content_description": "FlashStart doesn't require any DNS configuration neither in the firewall itself nor inside the clients. Before using FlashStart you need to",
+      "status": "Status",
+      "status_enabled": "Enabled",
+      "status_disabled": "Disabled",
+      "username": "Username",
+      "username_helper": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:571]:
```suggestion
      "username_helper": "Enter the username or the email address registered on https://flashstart.nethesis.it/.",
```

File: public/i18n/en/translation.json [hunk 2]
```diff
@@ -560,6 +560,24 @@
...
+      "password": "Password",
+      "zones": "Zones to filter",
+      "zones_placeholder": "Choose zones",
+      "sourcebypass": "Bypass source IPs or networks",
+      "sourcebypass_helper": "Lorem ipsum, or lipsum as it is sometimes known",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:576]:
```suggestion
      "sourcebypass_helper": "Enter IP addresses or networks in CIDR format. The addresses entered will not be subject to the DNS filter",
```

File: public/i18n/en/translation.json [hunk 3]
```diff
@@ -681,6 +681,25 @@
     "security": {
       "title": "Security"
     },
+    "flashstart": {
+      "title": "Flashstart filter",
```
Comment (andre8244) [inline public/i18n/en/translation.json:685]:
```suggestion
      "title": "FlashStart filter",
```

File: public/i18n/en/translation.json [hunk 4]
```diff
@@ -681,6 +681,25 @@
...
+      "status": "Status",
+      "status_enabled": "Enabled",
+      "status_disabled": "Disabled",
+      "username": "Username",
+      "username_helper": "Enter the username or the email address registered on https://flashstart.nethesis.it/.",
+      "password": "Password",
+      "zones": "Zones to filter",
+      "zones_placeholder": "Choose zones",
+      "sourcebypass": "Bypass source IPs or networks",
+      "sourcebypass_helper": "Enter IP addresses or networks in CIDR format. The addresses entered will not be subject to the DNS filter",
```
Comment (andre8244) [inline public/i18n/en/translation.json:698]:
```suggestion
      "sourcebypass_helper": "Enter IP addresses or networks in CIDR format. The addresses entered won't be subject to the DNS filter",
```

File: src/components/standalone/security/FlashstartContent.vue [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+      zones.value = Object.keys(getZones.data).map((key) => ({
+        id: getZones.data[key].name,
+        label: getZones.data[key].name
+      }))
+    }
+  } catch (exception: any) {
+    errorLoadingData.value.notificationTitle = t('error.cannot_retrieve_zones')
+    errorLoadingData.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  } finally {
+    await getConfiguration()
```
Comment (andre8244) [inline src/components/standalone/security/FlashstartContent.vue:80]: Let's invoke `list_zones` and `get-config` in parallel

**Review by andre8244 — APPROVED** [review]

**Pull request merged**